projects/core/src/lib/types/element-state.ts

```diff
@@ -0,0 +1,4 @@
+export interface ElementState {
```

Comment:
MaskitoElementState

Reply:
This interface is not Public API. Should we rename it ?

Reply:
Public API:
https://github.com/Tinkoff/maskito/blob/80384b30204eeacde1c7318efa36faf5cf477c5e/projects/core/src/index.ts#L1-L2

Reply:
as you wish))

Reply:
I think the shorter name is better)

The prefix `Maskito` don't add anything to the meaning of naming when you read code.

But public API should be all prefixed, of course!